syserol/figures/figure3.py

```diff
@@ -13,12 +13,40 @@ def makeFigure():
     ax, f = getSetup((9, 3), (1, 3))
     rep = 10
 
-    comps = np.arange(1, 11)
-    df = pd.concat([pd.DataFrame({'Components': comps, 'R2X': evaluate_missing(comps, 15, chords=True)[0]})
+    comps = np.arange(1, 6)
+
+    try:
+        chords_df = pd.read_csv('syserol/data/fig3_chords_df.csv')
```

Comment:
@murphymadeleine21 you want try-catch blocks building each of these *separately*. Right now this will build everything unless everything is saved, which puts you in exactly the same position you were in before.

Reply:
Made the change to help get this moving along.